src/main/java/com/openshift/internal/restclient/model/volume/VolumeSource.java

```diff
@@ -39,4 +39,35 @@ public String getName() {
 		return asString(getNode(), getPropertyKeys(), NAME);
 	}
 
+    @Override
+    public void setName(String name) {
+        set(getNode(), getPropertyKeys(), NAME, name);
+    }
+
+    @Override
+    public String toJSONString() {
+        if (!get(getNode(), getPropertyKeys(), NAME).isDefined()) {
```

Comment:
You can check with something like StringUtils.isBlank(getName()) since all the getters will return a value, specifically in this case it will return an emptystring which is bad given name is essentially the identity.